lib/client.js

```diff
@@ -263,7 +263,7 @@ extend(Raven.prototype, {
     kwargs.environment = kwargs.environment || this.environment;
     kwargs.logger = kwargs.logger || this.loggerName;
     kwargs.event_id = eventId;
-    kwargs.timestamp = new Date().toISOString().split('.')[0];
+    kwargs.timestamp = kwargs.timestamp || new Date().toISOString().split('.')[0];
```

Comment:
Can you mention this parameter inside `usage` docs in `Additional Data` section? Including desired timestamp format?